src/Microsoft.AspNet.Hosting/Internal/HostingLoggerExtensions.cs

```diff
@@ -189,7 +236,28 @@ public override string ToString()
             {
                 if (_cachedToString == null)
                 {
-                    _cachedToString = $"Request finished in {_elapsed.TotalMilliseconds}ms {_httpContext.Response.StatusCode} {_httpContext.Response.ContentType}";
+                    if (_logLevel <= LogLevel.Debug)
```

Comment:
@rynowak @lodejard what do you think? This avoids the duplication of data in multiple events, and keeps Information from getting too noisy. But I don't know if it makes sense to have an event that conditionally changes levels.